src/tests/electron/tests/automated-checks-view.test.ts

```diff
@@ -207,4 +208,67 @@ describe('AutomatedChecksView', () => {
             expect(boxStyle.height).toBeCloseTo(expectedHighlightBoxStyles[index].height);
         });
     }
+
+    const setupWindowForCommandBarReflowTest = async (
+        narrowFactor: number,
+    ): Promise<RawResult<any>> => {
+        await app.setFeatureFlag(UnifiedFeatureFlags.leftNavBar, true);
+
+        const width = narrowModeThresholds.collapseCommandBarThreshold - narrowFactor;
+
+        app.client.browserWindow.restore();
+        app.client.browserWindow.setSize(width, height);
+
+        // Note: the following call returns a different type of object than is specified
+        // by the typescript return type when the element is found
+        return automatedChecksView.client.$(AutomatedChecksViewSelectors.leftNavHamburgerButton);
+    };
+
+    it('command bar reflows when narrow mode threshold is crossed', async () => {
+        const result = await setupWindowForCommandBarReflowTest(2);
+        expect(result.value).not.toBeNull();
+    });
+
+    it('command bar does not reflow when narrow mode threshold is not crossed', async () => {
+        const result = await setupWindowForCommandBarReflowTest(0);
+        expect(result.value).toBeNull();
+    });
+
+    it('hamburger button click opens and closes left nav', async () => {
+        await setupWindowForCommandBarReflowTest(2);
+
+        const result = await automatedChecksView.client.$(
+            AutomatedChecksViewSelectors.fluentLeftNav,
+        );
+        expect(result.state).toBe('failure');
+
+        await automatedChecksView.client.click(AutomatedChecksViewSelectors.leftNavHamburgerButton);
+
+        // if the nav bar doesn't appear, the timeout has expired and the test fails.
+        await automatedChecksView.waitForSelector(AutomatedChecksViewSelectors.fluentLeftNav);
+
+        // The fluent left nav bar appears over the command bar
```

Comment:
I would write this as:

`Specifically clicks the hamburger button inside the fluent left nav (as there is one within the command bar as well)`